Pet-Clinic-Frontend/src/pages/ProfileEdit.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from 'react';
+import "./ProfileEdit.css";
+
+const backendUrl = import.meta.env.VITE_BACKEND_URL || 'http://localhost:8080';
+
+const ProfileEdit = () => {
+
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [error, setError] = useState<string | null>(null);
+  const [success, setSuccess] = useState<string | null>(null);
+
+
+  const sendToBackend = async (email: string, password: string) => {
+    try {
+      const response = await fetch(`${backendUrl}/profile-edit`, { // what endpoint is in backEnd??
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({ email, password }),
+      });
+
+      if (!response.status.toString().startsWith('2')) {
+        setError('Failed to update profile. Please try again.');
+      } else {
+        setSuccess('Profile updated successfully!');
+      }
+
+    } catch (error) {
+      console.error('Error:', error);
+      setError('An error occurred. Please try again later.');
+    }
+  };
```

Comment:
This function as is might be in a different module too 😉

Reply:
yes it is obvious. :-)